client_secret.json

```diff
@@ -0,0 +1 @@
+{"web":{"client_id":"417180537270-fkp5muapn4psf0nrn9kuod3dvlalc59r.apps.googleusercontent.com","project_id":"dark-hall-182220","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://accounts.google.com/o/oauth2/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"nzFCsRez3cq2sqhjbfa97hCY","redirect_uris":["http://localhost:8080/"],"javascript_origins":["http://localhost:8080"]}}
```

Comment:
Probably don't want to commit this as it might contain sensitive information (we can upload this file manually)